server/fileblame.go

```diff
@@ -462,6 +469,57 @@ func extendDiff(
 	return lines, content_lines, nil
 }
 
+var logPaginationLimit = 3
```

Comment:
Made this low to make it easier to test, increase it to 100 before landing.

Reply:
Yep, I do the same! :)